### PR TITLE
Added changes which are required  when upgraded to python 3.9

### PR DESCRIPTION
**Pull request by rutdeshm**

### CODE REVIEW

**Review by anarang — COMMENTED** [review]
just ran through the code, added some suggestions there, would let @noopurmath3 and @pbadak do the final review.

File: suites/test_sitemap.py [hunk 1]
```diff
@@ -81,17 +83,31 @@ def verify_assembly_sitemap(self, api_auth):
...
         lcc.log_info("Creating a dict for the sitemap endpoint responses for verification...")
 
-        for sitemap in root:
-            children = sitemap.getchildren()
-            xml_dict[children[0].text] = children[1].text
-
-        print(str(xml_dict))
+        #print(type(xml_dict))
+        #print(xml_dict)
+        #lcc.log_info(xml_dict.items())
```
[the remark this answers is not in the document]
Comment (rutdeshm) [reply suites/test_sitemap.py:92]: @anarang I will un-comment the that line.

File: suites/test_sitemap.py [hunk 2]
```diff
@@ -81,17 +83,31 @@ def verify_assembly_sitemap(self, api_auth):
...
+                if (k2 == "url"):
+                    for i in range(len(xml_dict[k1][k2])):
+                        if xml_dict[k1][k2][i]['loc'] == url_loc:
+                            publish_date = xml_dict[k1][k2][i]['lastmod']
+                            break
+
+        check_that("publish_date value assignment check ", publish_date, not_equal_to(""))
+
+
+
```
[the remark this answers is not in the document]
Comment (rutdeshm) [reply suites/test_sitemap.py:109]: @anarang Sure,I will configure pylint with pycharm.

File: suites/test_sitemap.py [hunk 3]
```diff
@@ -135,6 +151,37 @@ def verify_module_sitemap(self, api_auth):
...
         xml_dict = {}
+
+        print(url_loc)
+        print(type(response.text))
+        xml_dict_ord = xmltodict.parse(response.text)
+        xml_dict = json.loads(json.dumps(xml_dict_ord))
+        lcc.log_info("Creating a dict for the sitemap endpoint responses for verification...")
+        # lcc.log_info(root)
+
+        # for sitemap in root:
```
[the remark this answers is not in the document]
Comment (rutdeshm) [reply suites/test_sitemap.py:162]: @anarang ok,will do that

File: suites/test_sitemap.py [hunk 4]
```diff
@@ -135,6 +151,37 @@ def verify_module_sitemap(self, api_auth):
...
+                    for i in range(len(xml_dict[k1][k2])):
+                        if xml_dict[k1][k2][i]['loc'] == url_loc:
+                            publish_date = xml_dict[k1][k2][i]['lastmod']
+                            break
+
+        check_that("publish_date value assignment check ", publish_date, not_equal_to(""))
+
+
+
+        '''
```
[the remark this answers is not in the document]
Comment (rutdeshm) [reply suites/test_sitemap.py:184]: @anarang For commenting out the multiline code I have used that,as per above comment I will remove that code.

**Review by noopurmath3 — CHANGES_REQUESTED** [review]
@rutdeshm Please remove the .DS_Store file and add it to .gitignore file (I should not be commited)
And please add the config.ini.sample back

**Review by noopurmath3 — APPROVED** [review]
LGTM
However, going forward please try to add a unique message to each commit which summarizes the changes made in that particular commit

**Pull request merged**